pkg/scaffold/controller/controllertest.go

```diff
@@ -113,7 +114,12 @@ func TestReconcile(t *testing.T) {
 	defer close(StartTestManager(mgr, g))
 
 	// Create the {{ .Resource.Kind }} object and expect the Reconcile{{ if .Resource.CreateExampleReconcileBody }} and Deployment to be created{{ end }}
-	g.Expect(c.Create(context.TODO(), instance)).To(gomega.Succeed())
+	err = c.Create(context.TODO(), instance)
```

Comment:
context.Background() ?

Reply:
context.TODO() has been used multiple time in this test and many other files.
I guess it's better to deal with `context` stuff in a followup PR :)
Filed https://github.com/kubernetes-sigs/controller-tools/issues/29